models/attribute-definition/src/attribute-definition-draft/generator.ts

```diff
@@ -0,0 +1,28 @@
+import * as AttributeType from '@commercetools-test-data/attribute-type';
+import { LocalizedString } from '@commercetools-test-data/commons';
+import { fake, Generator } from '@commercetools-test-data/core';
+import { TAttributeDefinitionDraft } from './types';
+
+// https://docs.commercetools.com/api/projects/productTypes#attributedefinitiondraft
+
+const generator = Generator<TAttributeDefinitionDraft>({
+  fields: {
+    type: fake(() => AttributeType.random()),
+    name: fake((f) => f.commerce.productName()),
+    label: fake(() => LocalizedString.random()),
+    inputTip: fake(() => LocalizedString.random()),
+    isRequired: fake((f) => f.datatype.boolean()),
+    attributeConstraint: fake((f) =>
+      f.helpers.arrayElement([
+        'SameForAll',
+        'None',
+        'Unique',
+        'CombinationUnique',
```

Comment:
This should be a constant ([example](https://github.com/commercetools/test-data/blob/main/models/channel/src/constants.ts)), then you could do something like `f.helpers.arrayElement([Object.values(attributeConstraints)])` and we can later reference it as `attributeConstraints.SameForAll` in a preset.